components/homme/CMakeLists.txt

```diff
@@ -409,7 +411,7 @@ if(NOT BUILD_HOMME_WITHOUT_PIOLIBRARY)
     
     SET (CPRNC_INSTALL_DIR ${HOMME_BINARY_DIR}/utils/cime/tools/cprnc)
     SET (CPRNC_BINARY ${HOMME_BINARY_DIR}/utils/cime/tools/cprnc/cprnc)
-    ADD_SUBDIRECTORY(utils/cime/tools/cprnc)
```

Comment:
does this change belong to this PR?

Reply:
No, but you can't build Homme standalone without specifyin CPRNC_DIR without this mod. If you'd rather split into two PRs, I can certainly do that. I just thought this would speed up things.